.github/workflows/escalate.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         uses: atlassian/gajira-login@v3
         env:
           JIRA_BASE_URL: ${{ secrets.JIRA_BASE_URL }}
-          JIRA_USER_EMAIL: 'rgraber@edx.org'
+          JIRA_USER_EMAIL: 'productivitysysops@2u.com'
```

Comment:
Can you provide a link to any existing (or new docs) that explains what productivitysysops@2u.com is and how it should be used? Please update the PR description. Thanks.

Reply:
There are currently no such docs as the user is new and IT has not put much information around it

Reply:
If not a doc, I recommend we write at least a comment about where this email came from and how it is meant to be used. Could even just be a PR comment, but something to explain what you know now, but no one will know when they look at this in the future. The PR description is simply providing no context, including tickets/issues/etc.